static/app/views/issueDetails/streamline/eventList.tsx

```diff
@@ -75,17 +69,12 @@ export function EventList({group}: EventListProps) {
     eventView.sorts = [{field: 'timestamp', kind: 'desc'}];
   }
 
-  const grayText = css`
-    color: ${theme.subText};
-    font-weight: ${theme.fontWeightNormal};
-  `;
-
   const isRegressionIssue =
     group.issueType === IssueType.PERFORMANCE_DURATION_REGRESSION ||
     group.issueType === IssueType.PERFORMANCE_ENDPOINT_REGRESSION;
 
   return (
-    <StreamlineEventsTable>
+    <EventListTable pagination={{enabled: false}}>
```

Comment:
@scttcper It's a little better now, but since the EventsTable does it's own query, we only have access to the total count from inside the component, so it uses renderProps for the header.

I lifted as many styles as possible out into the separate table file to keep this cleaner and shared, let me know what you think.